test/Collateral.t.sol

```diff
@@ -267,6 +274,35 @@ contract CollateralContractTest is Test {
         );
     }
 
+    function testRevokeAuthorizedSigner() public {
+        depositCollateral(SENDER_ADDRESS, receiverAddress, COLLATERAL_AMOUNT);
+
+        authorizeSignerWithProof(SENDER_ADDRESS, authorizedSignerPrivateKeys[0], authorizedsigners[0]);
+        vm.prank(SENDER_ADDRESS);
+        collateralContract.thawSigner(authorizedsigners[0]);
+
```

Comment:
Perhaps could you test here that the indexer can still withdraw during the thawing of the signer?